<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QNote</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/ico/Dtafalonso-Yosemite-Flat-Notes.ico</normaloff>:/ico/Dtafalonso-Yosemite-Flat-Notes.ico</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widgetHeader" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>801</width>
         <height>52</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="hLayoutHeader">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnOpen">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border-image: url(:/ico/Pictogrammers-Material-Folder-Folder-outline.ico);}
QPushButton:hover { border-image: url(:/ico/Pictogrammers-Material-Folder-Folder-open-outline.ico); }
QPushButton:hover:pressed { border-image: url(:/ico/Pictogrammers-Material-Folder-Folder-open.ico);}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSave">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{ border-image: url(:/ico/Icons8-Windows-8-Programming-Save_u.ico);}
QPushButton:hover { border-image: url(:/ico/Icons8-Windows-8-Programming-Save-As.ico);}
QPushButton:hover:pressed {border-image: url(:/ico/Icons8-Windows-8-Programming-Save.ico);}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClear">
          <property name="minimumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border-image: url(:/ico/Ionic-Ionicons-Ban-outline.ico);}

QPushButton:hover {border-image: url(:/ico/Fa-Team-Fontawesome-FontAwesome-Ban.ico);}
QPushButton:hover:pressed {border-image: url(:/ico/Ionic-Ionicons-Ban-sharp.ico);}


</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClose">
          <property name="minimumSize">
           <size>
            <width>38</width>
            <height>38</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border-image: url(:/ico/Steve-Zondicons-Close-Outline.ico);}
QPushButton:hover {border-image: url(:/ico/Custom-Icon-Design-Mono-General-1-Close.ico);}
QPushButton:hover:pressed {border-image: url(:/ico/Steve-Zondicons-Close-Solid.ico);}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="hSpacer_header_middle">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnAbout">
          <property name="minimumSize">
           <size>
            <width>1</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border-image: url(:/ico/Bootstrap-Bootstrap-Bootstrap-exclamation-circle.ico);}
QPushButton:hover {border-image: url(:/ico/Custom-Icon-Design-Mono-General-about.ico);}
QPushButton:hover:pressed {border-image: url(:/ico/Bootstrap-Bootstrap-Bootstrap-exclamation-circle-fill.ico);}



</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>15</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="hSpacer_header_rigth">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QWidget" name="widgetBottom" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LayoutDirection::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 127);</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>700</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelPosition">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>第1行，第1列 </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
